docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -228,7 +234,7 @@ The above configuration properties are a minimal set to help you get started.
 Please see :doc:`/admin` and :doc:`/security` for a more comprehensive list.
 In particular, see :doc:`/admin/resource-groups` for configuring queuing policies.
 
-.. _log-levels:
+.. _log_properties:
 
 Log levels
```

Comment:
The heading should be called `Logging config` or `Logging properties` too?

Reply:
We could change it to Log configuration maybe .. I did not want to go into more changes. This page overall needs a lot more content and a break up into separate expanded pages